hashmap.c

```diff
@@ -50,6 +50,23 @@ unsigned int memihash(const void *buf, size_t len)
 	return hash;
 }
 
+/*
+ * Incoporate another chunk of data into a memihash
+ * computation.
+ */ 
+unsigned int memihash2(unsigned int hash_seed, const void *buf, size_t len)
```

Comment:
Good thinking!

I will probably rename this to `memihash_cont()` and rename the input parameter `hash_seed` to `hash` and then reuse it for the computation.